lib/saml_idp/attribute_decorator.rb

```diff
@@ -1,7 +1,7 @@
 require 'delegate'
 module SamlIdp
   class AttributeDecorator < SimpleDelegator
-    alias_method :source, :__getobj__
+    alias source __getobj__
```

Comment:
ditto for `alias_method`, I would see if we can flip that back

Reply:
fixed in https://github.com/18F/saml_idp/pull/98/commits/2b8c1afa357e64a05ddf8932dfae209da2a40a6a